llama_hub/tools/slack/base.py

```diff
@@ -13,10 +13,30 @@
 logger = logging.getLogger(__name__)
 
 
+class DataLoaderOutput(BaseModel):
+    result: List[Document]
+
+
+class MessageSenderOutput(BaseModel):
+    # TODO: Once we have a Pydantic definition for `slack_sdk.web.SlackResponse`, we can make this more useful.
+    # Today, if we add `slack_response: SlackResponse` here, it will give RuntimeError:
+    # no validator found for <class 'slack_sdk.web.slack_response.SlackResponse'>, see `arbitrary_types_allowed` in Config
+    pass
+
+
+class ChannelFetcherOutput(BaseModel):
+    channels: List[str]
+
```

Comment:
Right, it indeed changes the returned types, but I can guarantee you nothing is broken, because the tools aren't working for anyone in the first place. 

With the current `main` branch code, as soon as an Agent attempts to iterate through all tools, this exception happens (the Agent doesn't even have to be intending to use the Slack tool):

<img width="799" alt="before" src="https://github.com/run-llama/llama-hub/assets/594058/0e9a455f-2e84-4803-8a0f-d939c94ba160">

Reply:
ah interesting. Okay thanks for this context. 🦙